src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2624,7 +2624,51 @@ void ApplicationManagerImpl::UnregisterApplication(
     } else {
       resume_controller().RemoveApplicationFromSaved(app_to_remove);
     }
+
+    application_manager::VehicleInfoSubscriptions vehicle_data_for_unsubscribe =
+        SelectVehicleDataForUnsubscribe(app_to_remove);
+
+    if (!vehicle_data_for_unsubscribe.empty()) {
+      auto message_to_hmi =
+          MessageHelper::CreateUnsubscribeVehicleDataMessageForHMI(
+              vehicle_data_for_unsubscribe, app_to_remove);
+
+#ifdef DEBUG
```

Comment:
@IGapchuk I would like to remove debug code from pull request to release branch

Reply:
@AKalinich-Luxoft  Removed in 11b04298dca9cf017301e7cf54e3b206cd43d883.